--- ebtables-v2.0.10-4/ebtables.sysv.orig	2012-05-11 09:58:45.074144647 +0100
+++ ebtables-v2.0.10-4/ebtables.sysv	2012-05-11 09:59:54.905992629 +0100
@@ -11,6 +11,15 @@
 #
 # config: __SYSCONFIG__/ebtables         (text)
 #         __SYSCONFIG__/ebtables.<table> (binary)
+#
+### BEGIN INIT INFO
+# Provides: ebtables
+# Required-Start: $network
+# Required-Stop: $network
+# Default-Start: 3 4 5
+# Short-Description: Ethernet Bridge filter tables
+# Description: Ethernet Bridge filter tables
+### END INIT INFO
 
 source /etc/init.d/functions
 source /etc/sysconfig/network
